### PR TITLE
Add semantic html elements

### LINKED ISSUE #1: Add semantic HTML landmarks
Portfolio is not really accessible at the moment. Need to add semantic HTML e.g. header, main, nav etc.

### PR DESCRIPTION
**Pull request by Hariprasad19036**

### CODE REVIEW

**Review by sano2019 — CHANGES_REQUESTED** [review]
The addition of semantic html should not change the look and styling of the site. Currently my name is added as a header and I think the footer look also changed.

In addition, there is a new master branch so please pull that before doing changes :)

File: src/components/layout/Footer.jsx [hunk 1]
```diff
@@ -9,7 +9,7 @@ import {
 
 const Footer = () => {
   return (
-    <div className='footer'>
+    <footer>
```
Comment (EmmaDawsonDev) [inline src/components/layout/Footer.jsx]:
```suggestion
    <footer className=‘footer’>
```

File: src/components/layout/Navbar.jsx [hunk 2]
```diff
@@ -3,7 +3,7 @@ import { NavLink } from 'react-router-dom';
 
 const Navbar = () => {
   return (
-    <div className='navbar'>
+    <nav>
```
Comment (EmmaDawsonDev) [inline src/components/layout/Navbar.jsx]:
```suggestion
    <nav className=‘navbar’>
```

**Review by EmmaDawsonDev — CHANGES_REQUESTED** [review]
The elements are now semantic but there’s no need to remove the classes as that affects the styling. Please add all css back.

**Comment by Hariprasad19036** [comment]
> The addition of semantic html should not change the look and styling of the site. Currently my name is added as a header and I think the footer look also changed.
> 
> In addition, there is a new master branch so please pull that before doing changes :)

Do you want me to change the header text / style? If yes what shall I change it to?

**Comment by sano2019** [comment]
@Hariprasad19036 

Basically, there was already a <header> element in the site, the "Welcome" is wrapped in it. Therefore there is no need to have the header at the top you are creating.

The only thing you needed to do is change the <div className="footer"> and the same for the nav bar into their proper semantic html elements with the same styling. I think in a previous commit you also removed the styling for the footer as it looks different in the version you are working on than it does in the one running online. 

Also in the latest PR it seems you are using back ticks (´) for the class names, this is preventing me from building the app. Should use single quotes (') :)

**Comment by Hariprasad19036** [comment]
@sano2019 Made the specified changes. Kindly review.

**Comment by sano2019** [comment]
Thank you @Hariprasad19036 :) Accepted and merging now.

**Pull request merged**